node/endpoint-handler.js

```diff
@@ -89,16 +90,18 @@ TChannelEndpointHandler.prototype.handleArg1 = function handleArg1(req, buildRes
             'no such endpoint service=%j endpoint=%j',
             req.serviceName, name));
     } else if (handler.canStream) {
-        handler.call(self, req, buildResponse);
+        handler.handleRequest(req, buildResponse);
     } else if (req.streamed) {
-        self.bufferArg23(req, buildResponse, handler);
+        // And implicitly, !handler.canStream, meaning fragmented but
+        // reaccumulated:
+        self.withArg23(req, buildResponse, handler);
     } else {
         res = buildResponse({streamed: false});
-        handler.call(self, req, res, req.arg2, req.arg3);
+        handler.handleRequest(req, res, req.arg2, req.arg3);
     }
 };
 
-TChannelEndpointHandler.prototype.bufferArg23 = function bufferArg23(req, buildResponse, handler) {
+TChannelEndpointHandler.prototype.withArg23 = function withArg23(req, buildResponse, handler) {
```

Comment:
if you're going to go for "consistency" with req.withArg23... actually use req.withArg23